### ISSUE TITLE
[modern] improve error on relay-compiler

### ISSUE THREAD

**Issue by sibelius** [facebook/relay#1751]
It would be really helpful to throw in which file or component this error happened

```jsx
Error: FindGraphQLTags: Substitutions are not allowed in graphql tags.
  at invariant (app/node_modules/relay-compiler/bin/relay-compiler:4714:12)
at getGraphQLText (app/node_modules/relay-compiler/bin/relay-compiler:4664:47)
at app/node_modules/relay-compiler/bin/relay-compiler:4573:27
at Array.forEach (native)
at CallExpression (app/node_modules/relay-compiler/bin/relay-compiler:4568:31)
at visit (app/node_modules/relay-compiler/bin/relay-compiler:4721:6)
at traverse (app/node_modules/relay-compiler/bin/relay-compiler:4734:8)
at visit (app/node_modules/relay-compiler/bin/relay-compiler:4724:4)
at app/node_modules/relay-compiler/bin/relay-compiler:4738:12
at Array.forEach (native)
```

**Comment by sibelius** [comment]
figure it out this error happen in this fragment

relay-codemod couldn't handle this case

```jsx
viewer: graphql`
      fragment ProductEdit_viewer on Viewer {
        id
        ${AttributeSelectField.getFragment('viewer', { filterByBrand: true })}
      }
    `,
```

**Comment by wincent** [comment]
Thanks for this report! Should be fixed as part of #1765, for which @Yan-J has an incoming fix, so I'm going to close this one.
